proposals/infra/INF-0002-root-signature-in-clang.md

```diff
@@ -304,15 +440,113 @@ elimation has completed.
 
 ### Validations in Sema
 
-TODO
+#### All the values should be legal.
+
+Some of these values are covered by syntactical checks in Sema.
+
+List all here for reference during DXIL generation checks.
+
+##### ShaderVisibility
+- SHADER_VISIBILITY_ALL
+- SHADER_VISIBILITY_ALL
+- SHADER_VISIBILITY_VERTEX
+- SHADER_VISIBILITY_HULL
+- SHADER_VISIBILITY_DOMAIN
+- SHADER_VISIBILITY_GEOMETRY
+- SHADER_VISIBILITY_PIXEL
+- SHADER_VISIBILITY_AMPLIFICATION
+- SHADER_VISIBILITY_MESH
+
+##### ParameterType
+- CBV
+- SRV
+- UAV
+- DescriptorTable
+- Constants32Bit
```

Comment:
The terms used here don't match the terms used in the grammar.  I'm still unconvinced that there is any value at all in listing anything here that is just "can be parsed by the grammar".  The more interesting validations to list are the ones that are syntactically correct but have semantic errors.

Reply:
Fixed.